content/03.results.md

```diff
@@ -114,69 +121,73 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
 
 <!-- TODO: possibly explore gene mutations driving signature 8, specifically for meningiomas, and describe -->
 
-![**Mutational co-occurrence and signatures highlight key oncogenic drivers.** A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types (annotated from `cancer_group` if N >= 10 or `Other` if N < 10); B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). D, Chromothripsis frequency across pediatric brain tumors shown by `cancer_group` with N >= 3. E, Sina plots of RefSig signature weights for signatures 1, 3, 8, 11, 18, 19, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figure3.png?sanitize=true){#fig:Fig3 width="7in"}
+![**Mutational co-occurrence and signatures highlight key oncogenic drivers.** A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types, which are denotated as "Other" when there are fewer than 10 samples per grouping; B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). D, Chromothripsis frequency across pediatric brain tumors for all cancer groups with N >= 3 samples. E, Sina plots of RefSig signature weights for signatures 1, 3, 8, 11, 18, 19, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figure3.png?sanitize=true){#fig:Fig3 width="7in"}
 
 ### Transcriptomic Landscape of Pediatric Brain Tumors
 
 #### Prediction of _TP53_ oncogenicity and telomerase activity
 
-To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
+To understand the _TP53_ phenotype in each tumor, we applied a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
```

Comment:
note this was my only edit towards this section of the manuscript. This will be in my next round.

Reply:
I don't see anything from line 164-190, aside from this, is that correct?

Reply:
Yes that's it for this PR!